client/source/js/modules/model/calibration.html

```diff
@@ -1,158 +1,173 @@
 <div class="page rich">
 
-    <div class="page_subheader">
-        <div class="section grid">
-            <form name="CalibrationForm" ng-submit="simulate()">
-                <div class="2">
-                    <input class="txbox" type="number" placeholder="Start year" ng-model="simulationOptions.startyear">
-                </div>
-                <div class="2">
-                    <input class="txbox" type="number" name="endYear" placeholder="End year" ng-model="simulationOptions.endyear" more-than="G.dataend">
-                    <span ng-show="CalibrationForm.endYear.$error.moreThan" class="error-hint">{{reportDataEndError()}}</span>
-                </div>
-                <div class="2">
-                    <button class="btn" ng-click="simulate()" ng-disabled="CalibrationForm.$invalid">View simulation</button>
-                </div>
-            </form>
-        </div>
-        <div class="section grid">
-
-            <label class="checkbox-inline">
-                <input type="checkbox" ng-disabled="!canDoFitting" ng-model="enableAutomaticCalibration">
-                Automatic calibration
-            </label>
-
-            <label class="checkbox-inline">
-                <input type="checkbox" ng-disabled="!canDoFitting" ng-model="enableManualCalibration">
-                Manual calibration
-            </label>
-            <span ng-show="!canDoFitting" class="error-hint">Run "View Simulation" in order to enable data calibration</span>
```

Comment:
I am not sure if we should go down this road: we might end up with a messy design if different screens will offer the same functionality.